client/consensus/aura/src/lib.rs

```diff
@@ -643,7 +648,10 @@ impl<B: BlockT, C, P, CAW> Verifier<B> for AuraVerifier<C, P, CAW> where
 				}
 
 				trace!(target: "aura", "Checked {:?}; importing.", pre_header);
-				telemetry!(CONSENSUS_TRACE; "aura.checked_and_importing"; "pre_header" => ?pre_header);
+				telemetry!(
+					self.client.telemetry(); CONSENSUS_TRACE; "aura.checked_and_importing";
```

Comment:
Same

Reply:
Done in c7139fd057 and 993040b3f3

oops i edited your comment XD